dist/calendar/CalendarDate.js

```diff
@@ -78,23 +78,29 @@ var CalendarDate = _react2['default'].createClass({
       mouseDown: false
     };
   },
+  componentWillUnmount: function componentWillUnmount() {
+    this.isUnmounted = true;
```

Comment:
if we do not, it will try to setState on an unmounted component throwing an error, this prevents that case